cmd/mic/main.go

```diff
@@ -18,16 +18,27 @@ var (
 	forceNamespaced   bool
 	versionInfo       bool
 	syncRetryDuration time.Duration
+	leaderElectionCfg mic.LeaderElectionConfig
 )
 
 func main() {
 	defer glog.Flush()
+	hostName, err := os.Hostname()
+	if err != nil {
+		glog.Fatalf("Get hostname failure.")
+	}
 	flag.StringVar(&kubeconfig, "kubeconfig", "", "Path to the kube config")
 	flag.StringVar(&cloudconfig, "cloudconfig", "", "Path to cloud config e.g. Azure.json file")
 	flag.BoolVar(&forceNamespaced, "forceNamespaced", false, "Forces namespaced identities, binding, and assignment")
 	flag.BoolVar(&versionInfo, "version", false, "Prints the version information")
 	flag.DurationVar(&syncRetryDuration, "syncRetryDuration", 3600*time.Second, "The interval in seconds at which sync loop should periodically check for errors and reconcile.")
 
+	// Leader election parameters
+	flag.StringVar(&leaderElectionCfg.ResourceName, "leader-election-name", hostName, "leader name. default is 'hostname'")
+	flag.StringVar(&leaderElectionCfg.Namespace, "leader-election-namespace", "default", "name space to create leader election objects. default is 'default' namesapce")
```

Comment:
We don't need to explicitly say the default value in the usage. Since the default value is defined, pflag will show that in the output.

For instance - 
```
  -syncRetryDuration duration
    	The interval in seconds at which sync loop should periodically check for errors and reconcile. (default 1h0m0s)
```

Reply:
Done